File: src/daemon/migrator/pkg/xml_reader/entities/review.go
package entities

type Review struct {
	Id                string `xml:"id,attr"`
	TasterId          string `xml:"taster_id,attr"`
	WineId            string `xml:"wine_id,attr" json:"wineId"`
	Points            int    `xml:"points,attr" json:"points"`
	ReviewDescription string `xml:"Review>ReviewDescription" json:"description"`
	TwitterHandle     string `json:"twitterHandle"`
	WineTitle         string
}

type ReviewDescription struct {
	Text string `xml:",chardata"`
}

File: src/daemon/watcher/lib/xml_parser/entities/review.ex
defmodule XmlParser.Entities.Review do
  @type t :: %__MODULE__{
          points: float(),
          twitter_handle: String.t(),
          wine_title: String.t(),
          description: String.t()
        }

  defstruct [:points, :twitter_handle, :wine_title, :description]
end
